configuration/configuration_test.go

```diff
@@ -193,6 +195,22 @@ func TestGetKeycloakEndpointUserinfo(t *testing.T) {
 	assert.NotNil(t, configurationData.GetKeycloakEndpointUserinfo())
 }
 
+func TestMultipleConfigurations(t *testing.T) {
+	t.Parallel()
+
+	configurationData1, err := configuration.NewConfigurationData(tEnvironmentVariableValueConfigFile)
+	require.NotNil(t, configurationData1)
+	require.Nil(t, err)
+
+	configurationData2, err := configuration.NewConfigurationData("")
+	require.NotNil(t, configurationData2)
+	require.Nil(t, err)
+
+	assert.NotEqual(t, configurationData1.GetPostgresHost, configurationData2.GetPostgresHost)
```

Comment:
Correct, they are coming from different configuration files, hence the `assert.NotEqual` .